katas/001_optimize_a_query.md

```diff
@@ -20,9 +20,9 @@ Since you have already gone through the Movie Demo setup, we'll need to set up a
 
 Go ahead and start a new case by clicking _Relevancy Cases_ drop down and choosing _Create a Case_.  
 
-Let's call the case _Notebook Computers_.   Then, instead of the default Solr instance, let's go ahead and use our Chorus Electronics index using this URL (notice the embedded user credentials):
+Let's call the case _Notebook Computers_.   Then, instead of the default Solr instance, let's go ahead and use our Chorus Electronics index using this URL:
 
-`http://solr:SolrRocks@localhost:8983/solr/ecommerce/select`
```

Comment:
Yea this only breaks in Chrome - https://chromestatus.com/feature/5669008342777856 <- blocks quepid queries with embedded credentials without any details. Firefox worked without issues.

Reply:
Would you be interested in documenting this in https://github.com/o19s/quepid/wiki/Troubleshooting-Solr-and-Quepid#solr-w-basic-auth as an issue?    Longer term, I'm both looking at A) adding proper CORS support to Solr 9, to get away from JSONP, and B) adding a proxy end point to Quepid so you can avoid doing a browser request...

Reply:
Good idea! Added a sentence to the wiki for documentnation:

> Be aware of the fact that browsers behave differently. Using Chrome may block Quepid queries with embedded credentials without any details. See https://chromestatus.com/feature/5669008342777856 for reference. On the other hand, Firefox may work without issues.